hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/autocleanup/CleanupTask.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2018 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.repository.jpa.autocleanup;
+
+/**
+ * Interface modeling a cleanup task.
+ */
+public interface CleanupTask extends Runnable {
+
+    /**
+     * Executes the cleanup task.
+     */
+    @Override
+    void run();
```

Comment:
If you have to overwrite and document the only inherited method from the super interface maybe the hierarchy is wrong 😉

Reply:
I think I disagree. The method is only overridden to clarify its semantics in the context of the sub-interface.